tests/bootloader_test/Makefile

```diff
@@ -0,0 +1,28 @@
+# name of your application
+APPLICATION = bootloader_test
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= samr21-xpro
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+# Comment this out to disable code in RIOT that does safety checking
+# which is not needed in a production environment but helps in the
+# development process:
+CFLAGS += -DDEVELHELP
+
+# Change this to 0 show compiler invocation lines by default:
+QUIET ?= 1
+
+USEMODULE += firmware
+
+APP_ID = 0xBB8
+APP_VERSION = 0x1
+
+BOARD_WHITELIST := samr21-xpro iotlab-m3 fox iotlab-a8-m3
```

Comment:
Well, I think this feature can also belong to the board, since some development boards come with a pre-flashed bootloader, which I think we don't aim to overwrite (until we have a nice solution to flash all the boards using this bootloader through serial or USB), so maybe is better to put it as a board feature.